eye2bids/edf2bids.py

```diff
@@ -241,6 +238,38 @@ def _extract_TaskName(events: list[str]) -> str:
     )
 
 
+def _extract_StartTime(events: list[str]) -> int:
+    StartTime = (
+        np.array(pd.DataFrame([st.split() for st in events if st.startswith("START")])[1])
+        .astype(int)
+        .tolist()
+    )
+    if len(StartTime) > 1:
+        e2b_log.info(
```

Comment:
```suggestion
        e2b_log.warning(
```

Reply:
I think we may to just use `warning.warns` instead of using the logger but we can decide on that later